models/public/colorization-v2-norebal/model.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+    The colorization-v2-norebal model is one of the [colorization](https://arxiv.org/pdf/1603.08511)
+    group of models designed to perform image colorization. For details
+    about this family of models, check out the [repository](https://github.com/richzhang/colorization).
+
+    The gray-scale or BGR image should be converted to LAB color space first.
+    The model input is a blob that consists of a L-channel of image of 1x1x224x224.
+    The mean values need to be subtracted as follows: 50.0
+    before passing the image blob into the network.
+
+    The model output is a blob that consists of AB channels of image of 1x2x56x56.
```

Comment:
Update description according to description in documentation.

Reply:
@eizamaliev corrected